src/main/java/com/osiris/autoplug/client/configs/SSHConfig.java

```diff
@@ -90,15 +90,18 @@ public SSHConfig() throws IOException, DuplicateKeyException, YamlReaderExceptio
                 "The file must be in the OpenSSH format.",
                 "Create keys with 'ssh-keygen -t rsa -b 4096' and add them to the file.",
                 "The genereated file can be found in the .ssh directory of the user that created the key, unless a different path was specified.",
-                "The generated file will be a file with no extension, which contains the private key.");
+                "The generated file will be a file with no extension, which contains the private key.",
+                "In the same directory as the private key, there will also need to be a file with the same name and a .pub extension, which contains the public key.");
 
         username = put(name, "username").setDefValues("autoplug")
-            .setComments("The username required to connect to the SSH console.");
+            .setComments(
+                "The username required to connect to the SSH console.",
+                "This username must be unique and not used by any other SSH-based services on the host machine.");
 
         password = put(name, "password")
             .setComments(
-                "The password is not encrypted and can be seen in plain text in this file.",
-                "It is recommended to change it to a secure password.");
+                "WARNING: The password is not encrypted and can be seen in plain text in this file.",
+                "For this reason, it is recommended to use a public key instead of a password for authentication.");
```

Comment:
I'm not an SSH pro, but isn't the key pair used? Since a public key is literally public u can't use that as password.

Reply:
true ill rephrase